projects/angular/src/data/datagrid/datagrid-page-size.spec.ts

```diff
@@ -25,6 +25,12 @@ export default function (): void {
         context.detectChanges();
         expect(context.clarityDirective.pageSizeOptions).toEqual([10, 20, 50, 100]);
       });
+
+      it('receives an input for page size options id', function () {
```

Comment:
do we need this test here if we have it below?

Reply:
Yeah. The pattern seems to be 1) a test that the input binding itself works and 2) a test for the behavior that sets the property directly. That way, we are effectively testing that it works either way the user sets the property.